tests/e2e/e2e_workflow.spec.js

```diff
@@ -33,15 +33,6 @@ describe('Basic end-to-end Workflow', function () {
 
   after(conf.restore);
 
-  var testArgs = ['test', '--watch', 'false'];
```

Comment:
This line still needs to stay though. CI fails because it's missing, and it's what makes unit tests run in single run mode.